src/control/server/drpc.go

```diff
@@ -142,9 +142,9 @@ func checkDrpcResponse(drpcResp *drpc.Response) error {
 	return nil
 }
 
-// newDrpcCall creates a new drpc Call instance for specified module, with
+// newDrpcCall creates a new drpc Call instance for specified with
 // the protobuf message marshalled in the body
-func newDrpcCall(module int32, method int32, bodyMessage proto.Message) (*drpc.Call, error) {
+func newDrpcCall(method drpc.Method, bodyMessage proto.Message) (*drpc.Call, error) {
```

Comment:
the focus of the changes is to avoid passing generic datatypes around (int32) as that has the potential to be misused and result in unexpected behaviour. with specific static types for each method much of those risks are mitigated.